monitor/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM alpine:3.15.0
+LABEL maintainer "Arulraj V <me@arulraj.net>"
+
+RUN apk add --no-cache --update \
+      tzdata \
+      bash \
+      mysql-client \
```

Comment:
@vijayabharathib most of these packages for backup and compressing. since we use only aws commends for this only awscli is needed.

Reply:
removed mysql client and gzip